### ISSUE TITLE
Cannot set property threads of #<Object> which has only a getter

### ISSUE THREAD

**Issue by arthurkirkosa** [vuejs/vuex#255]
I keep getting this error and can't seam to find a way to resolve it

I've uploaded my whole (tiny) project to [dropbox](https://www.dropbox.com/s/xe64ywpvcgtdxor/chat.zip?dl=0) if anyone cares to have a stab at it.

I've tried everything I know. 

Most of the files are from the chat example;

**Comment by ktsn** [comment]
Quick fix is to modify the store definition to following.

```js
export default new Vuex.Store({
  state,
  getters: Object.assign({}, getters),
  actions,
  mutations,
});
```

`laravel-elixir-webpack-official` uses webpack 2.0 internally and it exports modules as readonly objects.
In addition, Vuex will mutate the root `getters` object, then the error is happened.

This seems to be fixed by #253

**Comment by arthurkirkosa** [comment]
I've ended up having the getters object in `store.js`; as soon as the new code is tagged, I'll try it and let you know if it fixes my issue

**Comment by doabit** [comment]
Have the same issue with `vuex` next branch and `webpack` 2.

**Comment by ktsn** [comment]
It's fixed on 2.0.0-rc.4

**Comment by arthurkirkosa** [comment]
I can confirm 2.0.0-rc.4 fixed the issue
